stdlib/jms/src/main/ballerina/jms/durable_topic_subscriber.bal

```diff
@@ -30,12 +30,12 @@ public type DurableTopicSubscriber object {
     # + c - Configurations for a durable topic subscriber
     public function init(DurableTopicSubscriberEndpointConfiguration c) {
         self.config = c;
-        match (c.session) {
-            Session s => {
-                self.createSubscriber(s, c.messageSelector);
-                log:printInfo("Durable subscriber created for topic " + c.topicPattern);
-            }
-            () => {}
+        var session = c.session;
+        if (session is Session){
+             self.createSubscriber(session, c.messageSelector);
+             log:printInfo("Durable subscriber created for topic " + c.topicPattern);
+        } else {
```

Comment:
Can't we just get rid of the else block? Or add some log? Applies to other places as well.